waku/waku_archive/driver/postgres_driver/postgres_driver.nim

```diff
@@ -147,10 +151,16 @@ proc rowCallbackImpl(pqResult: ptr PGresult,
     wakuMessage.contentTopic = contentTopic
     wakuMessage.payload = @(payload.toOpenArrayByte(0, payload.high))
 
+    var msgHash: WakuMessageHash
+    let byteCount = copyFrom(msgHash, hashHex.toOpenArrayByte(0, 31))
+    assert byteCount == 32
```

Comment:
This seems quite similar as the following , present at `waku_archive/common.nim`. Maybe we can make use it somehow:
```
proc fromBytes*(T: type MessageDigest, src: seq[byte]): T =
```